src/app/create-account-page/create-account-page.component.html

```diff
@@ -0,0 +1,46 @@
+<div id="page-content">
```

Comment:
The text boxes show the warning as soon as one is unfocused. I think it may be better to call the validate methods only on `onSubmit()`. That would provide better feedback to the user when they click "Create Account" also (right now it does nothing when the forms aren't entered correctly).

Reply:
Yeah, I did it that way on purpose. I figured that If someone focused on a field and then unfocused, for instance if they were navigating the fields via the TAB key, it would remind them, that they forgot to fill out the field.

Reply:
Yeah, I see what you're saying. I'm good with it as you've made it.

Probably the ideal solution (but more effort than it's worth) in my mind would be to have the First Name and Last Name boxes only warn on Submit, the password box only warn on submit and when unfocused with more than 0 and less than 8 characters, and the email box warn on submit and when unfocused.

Reply:
Good idea. I've changed it to implement that functionality.